### DISCUSSION TITLE
Add event to control pages via knob (SD+)

### DISCUSSION THREAD

**Discussion by nasteka** [elgatosf/elgato#70] [Ideas]
My wife recently got SD+ and she is having trouble with switching pages. She thought that it would be handy to control pages with knobs, but we don't have such official plugins or events in the SDK to create this by our own.

I'm not sure if I should submit a request to the SDK project or if I should place a request for Stream Deck default plugins.

Anyway, from her feedback about the touch bar, it's useful, but when you're on a stream with an audience, you just lose control: you can accidentally press "mute" on touch section; you can try to swipe two or three times without success, and end up shifting your sight to SD to switch the page properly. Navigation with buttons isn't best option, when you have knobs.

Also the need to transfer such simple actions from buttons to knobs sounds reasaonbly, you don't need an icon for that, you just need the tactile feedback that the knobs already have. On the other hand, moving actions perfectly designed for knobs outside of knobs and taking up at least two buttons (prev, next) on each page doesn't sound user-friendly.

In the end, my wife and I will be really grateful for the event that could control pages with knobs, as a developer I'll make plugin for it the very next day after release :)

**Comment by ChekTek** [comment]
Hi @nasteka,

This is a great suggestion! I will relay this feedback to our Stream Deck team to talk about what this could look like.

**Comment by ChekTek** [comment]
Closing this as the team is aware of the request.
